README.md

```diff
@@ -0,0 +1,104 @@
+# setup-nomad-pack
+
+The `hashicorp/setup-nomad-pack` Action sets up the `nomad-pack` CLI in your GitHub Actions workflow by adding the binary to `PATH`.
+
+## Table of Contents
+
+- [setup-nomad-pack](#setup-nomad-pack)
+  - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Usage](#usage)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Requirements
+
+This GitHub Actions supports all commands that are available in the `nomad-pack` CLI.
+
+The `run`, `destroy`, `info`, and `status` commands require access to a Nomad cluster, as defined through the environment variable `NOMAD_ADDR` (and optionally: `NOMAD_TOKEN`).
+
+## Usage
+
+1.) Create GitHub Actions Secrets by going to the repository's _Settings_ tab, followed by expanding the _Secrets_ sidebar, and finally _Actions_.
+
+- Set the `NOMAD_ADDR` to the IP-address or hostname of a Nomad cluster that is routable for GitHub Actions Runners.
+- Set the `NOMAD_TOKEN` to a token with appropriate permissions to carry out Pack-specific operations on a Nomad cluster.
+
+> **Warning**
+> Running services such as Nomad on a publicly accessible port without authentication is a decidedly bad idea.
+>
+> Consult with your security team to define an access policy that meets your organization's security demands.
+
+<small>GitHub Actions run on a publicly-known list of [IP addresses](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#ip-addresses).
+This data may be retrieved through [HashiCorp Terraform](https://terraform.io/), using the [ip_ranges data source](https://registry.terraform.io/providers/integrations/github/latest/docs/data-sources/ip_ranges), allowing you to make IP-address _one_ of the security considerations.</small>
+
+2.) Create a GitHub Actions Workflow file (e.g.: `.github/workflows/nomad-pack.yml):
+
+```yaml
+name: nomad-pack
+
+on:
+  - push
+
+jobs:
+  setup-nomad-pack:
+    runs-on: ubuntu-latest
+    name: Run Nomad Pack
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup `nomad-pack`
+        # TODO: define `v1`
+        uses: hashicorp/setup-nomad-pack@v1
+        id: setup
+        with:
+          version: "0.0.1-techpreview2"
```

Comment:
Is is possible to parameterize this value so you can try different target versions more easily?

Reply:
Yes, using `workflow_dispatch`. I'll add this in a future PR!